hooks/open-telemetry/src/main/java/dev/openfeature/contrib/hooks/otel/MetricHookOptions.java

```diff
@@ -30,4 +30,10 @@ public class MetricHookOptions {
      */
     @Builder.Default
     private final List<DimensionDescription> setDimensions = Collections.emptyList();
+
+    /**
+     * Extra attributes added at hook constructor time.
+     */
+    @Builder.Default
+    private Attributes extraAttributes = Attributes.empty();
```

Comment:
Sorry for being late. How about `extraAttributes -> staticAttributes or defaultAttributes `?